packages/ringcentral-widgets/components/ContactDetails/i18n/es-419.js

```diff
@@ -1,9 +1,10 @@
 import presenceStatus from 'ringcentral-integration/modules/Presence/presenceStatus';
 import dndStatus from 'ringcentral-integration/modules/Presence/dndStatus';
+import phoneTypes from '../../../enums/phoneTypes';
 
 export default {
-  extensionLabel: "Ext.",
-  directLabel: "Directo",
+  [phoneTypes.extension]: "Ext.",
+  [phoneTypes.direct]: "Directo",
```

Comment:
The related annotation data should be updated also.

Reply:
it's testing while I thought just need to update `direct` and `ext`. An export and import will auto fix this right？